CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ FEATURES:
 * **New Data Source:** `huaweicloud_kms_key_v1`
 * **New Data Source:** `huaweicloud_kms_data_key_v1`
 * **New Data Source:** `huaweicloud_rds_flavors_v1`
+* **New Data Source:** `huaweicloud_iam_role_v3`
```

Comment:
v1.0.0 is already released! this should be added under v1.0.1 above.

Reply:
Done